src/DBase/dbaseDlg.cpp

```diff
@@ -202,7 +202,15 @@ void DBaseDlg::loadGraspButton_clicked(){
 	mCurrentFrame = 0;
 	//get new grasps from database manager
 	PROF_START_TIMER(GET_GRASPS_CALL);
-	if(!mDBMgr->GetGrasps(*mCurrentLoadedModel,hand->getDBName().toStdString(), &mGraspList)){
+
+	if (hand->getDBName() == NULL) {
```

Comment:
This hard-codes one specific convention of using handname_material which is specific to CGDB but others might not use. I would prefer not to do this.